### PR TITLE
[WIP] Fix handling of return code for system() calls

### PR DESCRIPTION
**Pull request by jkotas**
The value returned from system() is encoded bag of bits that gets misinterpretted once it starts propagate through the system. Normalize it to 0/1.

### CODE REVIEW

**Comment by jkotas** [comment]
@dotnet-bot test Windows_NT x64 Checked corefx_baseline
@dotnet-bot test Ubuntu x64 Checked corefx_baseline

**Comment by jkotas** [comment]
@dotnet-bot test Ubuntu x64 Checked corefx_baseline

**Comment by jkotas** [comment]
@dotnet-bot test Ubuntu x64 Checked corefx_baseline

File: tests/scripts/run-corefx-tests.py [hunk 1]
```diff
@@ -238,7 +238,7 @@ def main(args):
     log(command)
     returncode = 0 if testing else os.system(command)
     if not returncode == 0:
```
Comment (stephentoub) [inline tests/scripts/run-corefx-tests.py]: Is there a functional difference between this "not returncode == 0" and the "return code != 0" used several places below?

**Comment by jkotas** [comment]
@dotnet-bot test Ubuntu x64 Checked corefx_baseline

**Comment by jkotas** [comment]
Fixed #16063

**Pull request merged**